fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/ReadableWorkbook.java

```diff
@@ -49,7 +49,8 @@ private ReadableWorkbook(OPCPackage pkg) throws IOException {
 
         try {
             this.pkg = pkg;
-            sst = SST.fromInputStream(pkg.getSharedStrings());
+            InputStream is = pkg.getSharedStrings();
+            sst = is == null ? null : SST.fromInputStream(is);
```

Comment:
`null` `InputStream` is already handled in [`SST.fromInputStream()`](https://github.com/dhatim/fastexcel/blob/master/fastexcel-reader/src/main/java/org/dhatim/fastexcel/reader/SST.java#L25)

Reply:
Oh right

Reply:
Opened #148 to revert this